### PR TITLE
Adding the Xamarin Sample App

### PR DESCRIPTION
**Pull request by ejmgithub**

### CODE REVIEW

File: WebViewApp-Xamarin/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<Project>
+    <PropertyGroup>
+        <ApplicationId>com.ermer.testapp</ApplicationId>
```
Comment (lexi-msft) [inline WebViewApp-Xamarin/Directory.Build.props:3]: Nope

Comment (ejmgithub) [reply WebViewApp-Xamarin/Directory.Build.props]: Removed

**Review by lexi-msft — COMMENTED** [review]
Who owns that image? Did we create it? Are we allowed to use it?

File: WebViewApp-Xamarin/Xamarin/XamarinTunnel/XamarinTunnel.iOS/Info.plist [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		<string>microsoft-edge-https-intunemam</string>
+		<string>zips</string>
+		<string>ms-outlook</string>
+		<string>ms-outlook-intunemam</string>
+		<string>smart-ns</string>
+		<string>https-intunemam</string>
+		<string>http-intunemam</string>
+	</array>
+	<key>MinimumOSVersion</key>
+	<string>14.0</string>
```
Comment (lexi-msft) [inline WebViewApp-Xamarin/Xamarin/XamarinTunnel/XamarinTunnel.iOS/Info.plist:41]: ReadMe is set this to 15.0

**Pull request closed without merging**